src/csmake.pl

```diff
@@ -53,6 +53,18 @@
 }
 $instdir = abs_path($instdir);
 
+my $extract_targets = 0;
+
+# Check CSMAKEFLAGS environment variable
```

Comment:
\# Expand command-line arguments to include the CSMAKEFLAGS environment variable

Reply:
Changed it.